src/main/java/base/BasePage.java

```diff
@@ -25,6 +27,8 @@ public BasePage(WebDriver driver, String relativeUrl) {
         this.relativeUrl = validateAndFormatRelativeUrl(relativeUrl);
         this.driver = driver;
 
+        js = (JavascriptExecutor) driver;
```

Comment:
What do you think about moving these methods to separated class? sometimes you need js executor in other methods body

Reply:
Agree that we could use it as a helper (maybe?) but for now, I think it's okay to leave it here